src/hum/humlib.cpp

```diff
@@ -1085,6 +1085,21 @@ double Convert::standardDeviation(const vector<double>& x) {
 }
 
 
+double Convert::standardDeviation(const vector<int>& x) {
+	double sum = 0.0;
+	for (int i=0; i<(int)x.size(); i++) {
+		sum += x[i];
+	}
+	double mean = sum / x.size();
```

Comment:
(Code style) std algorithms should be preferred instead of raw loops. In this case `std::accumulate` seems natural.